marc_to_solr/lib/traject_config.rb

```diff
@@ -285,11 +285,29 @@
   accumulator << record.end_date_from_008
 end
 
-# Alma:needs to change https://github.com/pulibrary/marc_liberation/issues/822
-to_field 'cataloged_tdt', extract_marc('959a') do |_record, accumulator|
-  accumulator[0] = Time.parse(accumulator[0]).utc.strftime("%Y-%m-%dT%H:%M:%SZ") unless accumulator[0].nil?
+# catalog_date https://github.com/pulibrary/marc_liberation/issues/926
+# Bibliographic Enrichment -> Create date subfield 950b
+# Physical Items Enrichment -> Create date subfield 876d
+# Electronic Inventory Enrichment -> Activation date subfield 951w
+to_field 'cataloged_tdt' do |record, accumulator|
+  extractor_doc_id = MarcExtractor.cached("001")
+  doc_id = extractor_doc_id.extract(record).first
+  extractor_876d = MarcExtractor.cached("876d")
+  extractor_951w = MarcExtractor.cached("951w")
+  extractor_950b = MarcExtractor.cached("950b")
```

Comment:
Maybe we should move these `MarcExtractor.cached` lines down into the conditional? It seems like we don't need need them all for each record, and it would be easier to see how they match up with the `extract` lines if they're together.

Reply:
I will move them yes. I had them initially in the conditional but I moved them up when I tried to refactor it. However I decided to keep the conditional as it is because I felt it to be more descriptive, at least in my mind, and didn't put them back in the conditional.